File: src/API/WesternStatesWater.WestDaat.Contracts.Client/Requests/Conservation/WaterConservationApplicationApprovalRequest.cs
using WesternStatesWater.WestDaat.Common.DataContracts;

namespace WesternStatesWater.WestDaat.Contracts.Client.Requests.Conservation;

public class WaterConservationApplicationApprovalRequest : ApplicationStoreRequestBase
{
    public Guid WaterConservationApplicationId { get; set; }
    
    public ApprovalDecision ApprovalDecision { get; set; }
    
    public string ApprovalNotes { get; set; }
}

File: src/DashboardUI/src/data-contracts/WaterConservationApplicationApprovalRequest.ts
import { ApplicationStoreRequestBase } from './ApplicationStoreRequestBase';
import { ApprovalDecision } from './ApprovalDecision';

export interface WaterConservationApplicationApprovalRequest extends ApplicationStoreRequestBase {
  $type: 'WaterConservationApplicationApprovalRequest';
  waterConservationApplicationId: string;
  approvalDecision: ApprovalDecision;
  approvalNotes: string;
}
